platforms/documentation/docs/src/docs/userguide/reference/third_party_integration.adoc

```diff
@@ -15,139 +15,4 @@
-This means that you can use the same API to work with builds that use different versions of Gradle, including versions that are newer or older than the version of the Tooling API that you are using.
-The Tooling API is Gradle wrapper aware and, by default, uses the same Gradle version as that used by the wrapper-powered build.
-
-Some features that the Tooling API provides:
-
-* Query the details of a build, including the project hierarchy and the project dependencies, external dependencies (including source and Javadoc jars), source directories and tasks of each project.
-* Execute a build and listen to stdout and stderr logging and progress messages (e.g. the messages shown in the 'status bar' when you run on the command line).
-* Execute a specific test class or test method.
-* Receive interesting events as a build executes, such as project configuration, task execution or test execution.
-* Cancel a build that is running.
-* Combine multiple separate Gradle builds into a single composite build.
-* The Tooling API can download and install the appropriate Gradle version, similar to the wrapper.
-* The implementation is lightweight, with only a small number of dependencies.
-It is also a well-behaved library, and makes no assumptions about your classloader structure or logging configuration.
-This makes the API easy to embed in your application.
-
-
-[[sec:embedding_daemon]]
-=== Tooling API and the Gradle Build Daemon
-
-The Tooling API always uses the Gradle daemon.
-This means that subsequent calls to the Tooling API, be it model building requests or task executing requests will be executed in the same long-living process.
-<<gradle_daemon.adoc#gradle_daemon,Gradle Daemon>> contains more details about the daemon, specifically information on situations when new daemons are forked.
-
-[[sec:embedding_quickstart]]
-=== Quickstart
-
-As the Tooling API is an interface for developers, the Javadoc is the main documentation for it.
-
-To use the Tooling API, add the following repository and dependency declarations to your build script:
-
-.Using the tooling API
-====
-include::sample[dir="snippets/toolingApi/runBuild/kotlin",files="build.gradle.kts[tags=use-tooling-api]"]
-include::sample[dir="snippets/toolingApi/runBuild/groovy",files="build.gradle[tags=use-tooling-api]"]
-====
-
-The main entry point to the Tooling API is the link:{javadocPath}/org/gradle/tooling/GradleConnector.html[GradleConnector].
-You can navigate from there to find code samples and explore the available Tooling API models.
-You can use link:{javadocPath}/org/gradle/tooling/GradleConnector.html#connect--[GradleConnector.connect()] to create a link:{javadocPath}/org/gradle/tooling/ProjectConnection.html[ProjectConnection].
-A `ProjectConnection` connects to a single Gradle project.
-Using the connection you can execute tasks, tests and retrieve models relative to this project.
-
-[[sec:embedding_compatibility]]
-=== Compatibility of Java and Gradle versions
```

Comment:
Should the content be merged into https://docs.gradle.org/current/userguide/compatibility.html#java